### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by Ef55**
Fixes:

- Creation of recurring events;
- Display of events' end time in calendar;
- Recurring events' instances not having the correct duration;
- The date picker not opening the correct month in event creator;
- Clean up the navigation bar (replace "sign" label, remove unused menus).

### CODE REVIEW

**Review by justagoodguy-omega — APPROVED** [review]
Niceuu

File: app/src/main/java/com/github/onedirection/database/queries/EventQueries.java [hunk 1]
```diff
@@ -146,15 +146,16 @@ public EventQueries(Database db) {
...
+        final long recurrenceLimit = newEventRecurrence.getEndTime().toEpochSecond();
+        final long refStartTime = event.getStartTime().toEpochSecond();
+        final long duration = event.getEndTime().toEpochSecond() - event.getStartTime().toEpochSecond();
 
         long x = (recurrenceLimit - refStartTime)/period;
         long tmpStartTime = refStartTime + x * period;
         while(x > 0) {
             Event newEvent = new Event(Id.generateRandom(), event.getName(), event.getLocationName(), event.getCoordinates(),
-                    TimeUtils.epochToZonedDateTime(tmpStartTime), TimeUtils.epochToZonedDateTime(tmpStartTime+period), Optional.of(newEventRecurrence));
+                    TimeUtils.epochToZonedDateTime(tmpStartTime), TimeUtils.epochToZonedDateTime(tmpStartTime+duration), Optional.of(newEventRecurrence));
```
Comment (Flechman) [inline app/src/main/java/com/github/onedirection/database/queries/EventQueries.java:158]: Thanks for finding this sneaky bug

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <string name="menu_calendar">Calendar</string>
     <string name="menu_map">Map</string>
     <string name="menu_account">Account</string>
-    <string name="menu_sign">Sign</string>
+    <string name="menu_sign">Login</string>
```
Comment (Flechman) [inline app/src/main/res/values/strings.xml:13]: Is it Sign In or Login ? THere's a view for Signing in, and another for Logging in.

**Review by Flechman — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Ef55) [reply app/src/main/res/values/strings.xml]: String changed.

**Review by Flechman — APPROVED** [review]

**Comment by Ef55** [comment]
Note: the coverage tanked quite severly, due to the deactivation of some menus. This should be back to normal next week with PR #213 (for event viewer) and the incoming GCalendar export (for account).

**Pull request merged**